commerce-theme-speedwell/commerce-theme-speedwell/src/css/components/_commerce_compare_table.scss

```diff
@@ -13,16 +13,41 @@
   tbody tr {
     height: auto;
   }
+	
+	.minium-card{
+		padding-bottom: 64px;
+		position: static;
 
-  .commerce-compare-table__title {
-    width: 150px;
-  }
+		&:hover {
+			box-shadow: none;
+		}
+
+		&__actions {
+			bottom: 20px;
+			left: 20px;
+			position: absolute;
+			right: 20px;
+		}
+	}
```

Comment:
Whitespace issues here. Indentation doesn't match surroundings, and missing a space in between `.minium-card` and `{`